core-bundle/src/DataContainer/ClipboardManager.php

```diff
@@ -47,7 +47,7 @@ public function get(string $table): array|null
         return $clipboard[$table];
     }
 
-    public function set(string $table, int|string|null $id, array|null $children, string $mode): void
+    public function set(string $table, int|string|null $id, array|string|null $children, string $mode): void
```

Comment:
```suggestion
    public function set(string $table, int|string|null $id, string|null $children, string $mode): void
```

I could not find any case where `$children` was an array. It was always either `null` or `'1'`.